pkg/controllers/provisioning/provisioner.go

```diff
@@ -174,14 +175,14 @@ func (p *Provisioner) consolidationWarnings(ctx context.Context, po *v1.Pod) {
 	if po.Spec.Affinity != nil && po.Spec.Affinity.PodAntiAffinity != nil {
 		if len(po.Spec.Affinity.PodAntiAffinity.PreferredDuringSchedulingIgnoredDuringExecution) != 0 {
 			if p.cm.HasChanged(string(po.UID), "pod-antiaffinity") {
-				logging.FromContext(ctx).Infof("pod %q has a preferred Anti-Affinity which can prevent consolidation", client.ObjectKeyFromObject(po))
+				log.FromContext(ctx).Info(fmt.Sprintf("pod %s has a preferred Anti-Affinity which can prevent consolidation", client.ObjectKeyFromObject(po)))
```

Comment:
Maybe I missed this, but why did we move it from %s to %q? Seems like we should keep it as is.

Reply:
Because otherwise we end up having unwanted back slashes. Example:
with `%q`
```terminal
{"level":"info","ts":"2024-05-20T22:05:56+03:00","msg":"pod \"ddd\" has a preferred Anti-Affinity which can prevent consolidation"}
```
with `%s`

```terminal
{"level":"info","ts":"2024-05-20T22:08:59+03:00","msg":"pod foo has a preferred Anti-Affinity which can prevent consolidation"}
```